src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/AdjournmentCalculateDateHelper.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.beanutils.BeanUtils;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Optional;
+import javax.validation.Valid;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+@Slf4j
+public final class AdjournmentCalculateDateHelper {
```

Comment:
Ideally this class would have its own dedicated test method, but due to the static nature of these classes that's not easy without a huge refactor.